### DISCUSSION TITLE
web development

### DISCUSSION THREAD

**Discussion by saimahendra282** [community/community#108135] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

can i use bootstrap/tailwind for html templates

**Comment by naveenalla3000** [comment, marked as answer]
yes! definitely. All the front-end developers are free to use their own confutable as framework for regularly css. The frameworks that supports are tailwindcss, bootstrap, shadcn and scss and many more in the ecosystem of js.
